.gitignore

```diff
@@ -13,3 +13,10 @@ tests/fixtures/drainpipe-test-project/*
 !.yarn/sdks
 !.yarn/versions
 .npmrc
+
+################
+# Editors / OS #
+################
+.idea
+.DS_Store
+.vscode
```

Comment:
@YesCT you might be interested in https://github.com/github/gitignore/tree/main?tab=readme-ov-file. It's generally recommended to put these in your global `.gitignore` so it automatically applies to all projects.

Reply:
Will do. And I'll pass on that feedback to our other projects where we add this.